packages/tools/src/utilities/rtstruct/RectangleROIStartEndThreshold.ts

```diff
@@ -0,0 +1,60 @@
+// comment
+class RectangleROIStartEndThreshold {
```

Comment:
I don't think you need this here, it should be in the adapters

Reply:
I don't know why we have any tools in the adapters package - this is required for the RectangleROIStartEndThresholdTool, which is definitely a tool, and belongs in the tools package.

Reply:
As agreed, this belongs in the tools package